build/KoreBuild.sh

```diff
@@ -69,6 +69,12 @@ if [ ! -d $sakeFolder ]; then
     mv "$toolsProject" "$toolsProject.norestore"
 fi
 
+nugetPath="$koreBuildFolder/nuget.exe"
+if [ ! -f $nugetPath ]; then
+    nugetUrl="https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
```

Comment:
I thought we didn't need to do this anymore....? Don't we use dotnet-restore instead of nuget.exe now?